pkg/piecestore/rpc/server/ttl/ttl.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package ttl
```

Comment:
this ttl db being separate is exactly what I had in mind. great job abstracting this out!

Reply:
👍